### PR TITLE
Pyband: refactor get_latest_block on client module

### LINKED ISSUE #2813: Patch `get_latest_block` on Client
Tasks
- [ ] Declare data class and return it instead of dict
- [ ] Add documentation

### PR DESCRIPTION
**Pull request by pzshine**
Fixed: #2813 

## Implementation details

- Refactored get_latest_block on client module
  - Wrote the test case
  - Created new type 
    - HexBytes
    - Timestamp
  - Created dataclass for Block
- Implemented parse_datetime

## Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [x] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [x] The pull request includes a description of the implementation/work done in detail.
- [x] The pull request includes any and all appropriate unit/integration tests
- [x] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

**Comment by pzshine** [comment]
Note: Will continue to fix the timezone thingy on the `test_parse_epoch_to_datetime`

File: helpers/pyband/pyband/utils.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from datetime import datetime, timezone
+
+
+def parse_datetime(raw_datetime) -> int:
+    splitted_datetime = raw_datetime.split(".")
```
Comment (sorawit) [inline helpers/pyband/pyband/utils.py]: `split_datetime`

File: helpers/pyband/pyband/utils.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from datetime import datetime, timezone
+
+
+def parse_datetime(raw_datetime) -> int:
```
Comment (sorawit) [inline helpers/pyband/pyband/utils.py]: `raw_datetime: str`

File: helpers/pyband/pyband/auth.py [hunk 3]
```diff
@@ -121,7 +121,9 @@ def is_reporter(self, validator: str, reporter_pubkey: str) -> bool:
 
         :return: True if the reporter is a registered reporter of the validator.
         """
-        reporter = PublicKey.from_acc_bech32(reporter_pubkey).to_address().to_acc_bech32()
+        reporter = (
```
Comment (sorawit) [inline helpers/pyband/pyband/auth.py]: let's agree on line length

**Review by sorawit — CHANGES_REQUESTED** [review]

File: helpers/pyband/pyband/auth.py (hunk 3 above)
Comment (pzshine) [reply helpers/pyband/pyband/auth.py]: We agreed with 119 line length and set it up to our vscode already

**Review by sorawit — APPROVED** [review]

**Review by evilpeach — APPROVED** [review]

**Pull request merged**